guides/common/modules/con_managing-dns-using-smart-proxies.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Managing_DNS_Using_Smart_Proxies_{context}"]
+= Managing DNS Using {SmartProxies}
+
+Your {SmartProxy} can manage DNS records.
+DNS management contains updating and removing DNS records from existing DNS zones.
```

Comment:
I was thinking about a generic bit of text. Something like:
```asciidoc
{Project} can manage DNS records using your {SmartProxy}.
DNS management contains updating and removing DNS records from existing DNS zones.
The {SmartProxy} has multiple providers, allowing you to integrate with your existing DNS infrastructure or deploy a new one.
```

A similar text can be used for the DHCP introduction.

Reply:
Applied and added.